packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -312,7 +357,10 @@ enum DependencyKind {
 /// This is the string we output next to each of our autogenerated transitive
 /// dependencies so that we can ignore them the next time we parse the
 /// pubspec.yaml file.
-const String kTransitiveMagicString = '# TRANSITIVE DEPENDENCY';
+const String kTransitiveMagicString= '# THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade';
```

Comment:
nit: missing a trailing `"` in the string, also, you have a space missing before the `=`.

Reply:
Fixed